src/index.html

```diff
@@ -22,6 +22,294 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        href="#"
+        class="header__link"
+      >
+        <img
+          src="images/logo.png"
+          alt="Moyo Logo"
+        />
+      </a>
+      <nav class="navigation">
+        <ul class="navigation__tabs">
+          <li class="navigation__tab">
+            <a
+              href="#"
+              class="navigation__link is-active"
+            >
+              apple
+            </a>
```

Comment:
You need to fix classes and code style
```suggestion
      <nav class="nav">
        <ul class="nav__list">
          <li class="nav__item">
            <a href="#" class="nav__link is-active">
              apple
            </a>
```